doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_gs.rst

```diff
@@ -43,45 +43,341 @@ On your computer, one of the following operating systems:
+
+         .. caution::
+            When working with west in the customer sampling release version of |NCS|, you must always use this shell window.
+
+      #. Run the following commands in your terminal to install the correct lxml dependency:
+
+         .. parsed-literal::
+            :class: highlight
+
+            pip uninstall -y lxml
+            pip install lxml
+
+      If you run into errors during the installation process, delete the :file:`.west` folder inside the :file:`ncs-lcs` directory, and start over.
+
+We recommend adding the nRF Util path to your environmental variables.
+
+.. _ug_nrf54h20_install_ncs:
+
+Installing the |NCS|
+====================
+
+After you have installed the toolchain, complete the following steps to get the customer sampling version of the |NCS|:
+
+1. In the terminal window opened during the toolchain installation, initialize west with the revision of the |NCS| from the customer sampling:
+
+   .. parsed-literal::
+      :class: highlight
+
+      west init -m https://github.com/nrfconnect/sdk-nrf --mr v2.6.99-cs2
+
+#. Enter the following command to clone the project repositories::
+
+      west update
+
+   Depending on your connection, this might take some time.
+
+#. Export a :ref:`Zephyr CMake package <zephyr:cmake_pkg>`.
+   This allows CMake to automatically load the boilerplate code required for building |NCS| applications::
+
+      west zephyr-export
+
+Your directory structure now looks similar to this::
+
+    ncs-lcs/work-dir
+    |___ .west
+    |___ bootloader
+    |___ modules
+    |___ nrf
+    |___ nrfxlib
+    |___ zephyr
+    |___ ...
+
+Note that there are additional folders, and that the structure might change.
+The full set of repositories and folders is defined in the manifest file.
+
+Installing the Terminal application
+***********************************
 
 On your computer, install `nRF Connect for Desktop`_.
 You must also install a terminal emulator, such as `nRF Connect Serial Terminal`_ (from the nRF Connect for Desktop application) or the nRF Terminal (part of the `nRF Connect for Visual Studio Code`_ extension).
 
+.. _ug_nrf54h20_gs_bringup:
+
+nRF54H20 bring-up
+*****************
+
+The following sections describe the steps required for the nRF54H20 bring up.
+
+.. _ug_nrf54h20_gs_fwbundle:
+
+Installing the nRF54H20 firmware bundle
+=======================================
+
+The nRF54H20 SoC requires the installation of a bundle ( :file:`nrf54h20_soc_binaries_v0.3.2.zip`) containing the precompiled firmware for the Secure Domain and System Controller:
+
+..
+   ### Update bundle link
+
+1. Download the `nRF54H20 firmware bundle`_.
+#. Move the :file:`ZIP` bundle to a folder of your choice, then indicate the bundle folder path to nRF Util using the following command::
+
+      nrfutil device x-provision-nrf54h --firmware *path-to_bundle_.zip_file* --serial-number *serialnumber*
```

Comment:
This command will flash secdomfw and sysctrlfw so it is described here: _Programming the SDFW and SCFW from the bundle_ 
I am not sure if the order make any difference but it should be done once. (Internal bringup says first BICR and then SDFW with SCFW)